packages/cli-server-api/src/devToolsMiddleware.ts

```diff
@@ -34,31 +29,49 @@ function launchDevTools(
   {host, port, watchFolders}: LaunchDevToolsOptions,
   isDebuggerConnected: () => boolean,
 ) {
+  const hostname = host ?? 'localhost';
```

Comment:
Seems to be very confusing using both host and hostname all around in this function. Typically hostname is a host name and it cannot contain port number while host contains both host name and port number.